sufia-models/app/models/sufia/works/work.rb

```diff
@@ -0,0 +1,7 @@
+# The base class of all works
```

Comment:
@awead Work is an abstract class. GenericWork is the simplest concrete type of Work.